pkg/build/strategies/sti/postexecutorstep.go

```diff
@@ -0,0 +1,459 @@
+package sti
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"syscall"
+
+	"github.com/openshift/source-to-image/pkg/api"
+	dockerpkg "github.com/openshift/source-to-image/pkg/docker"
+	"github.com/openshift/source-to-image/pkg/errors"
+	"github.com/openshift/source-to-image/pkg/tar"
+	"github.com/openshift/source-to-image/pkg/util"
+)
+
+type postExecutorStepContext struct {
+	// id of the previous image that we're holding because after committing the image, we'll lose it.
+	// Used only when build is incremental and RemovePreviousImage setting is enabled.
+	// See also: rememberPreviousImageStep and removePreviousImageStep
+	previousImageID string
+
+	// Container id that will be committed.
+	// See also: commitBuilderImageStep/commitRuntimeImageStep
+	containerID string
+
+	// See also: commitBuilderImageStep/commitRuntimeImageStep
+	location string
+
+	// Container id after committing.
+	// See also: commitBuilderImageStep/commitRuntimeImageStep and reportAboutSuccessStep
+	imageID string
+
+	// Labels that will be passed to a callback.
+	// It's labels that is used during commit of builder image (or runtime image if it's used).
```

Comment:
sorry my question was intended as a suggestion.
"// these labels are added to the builder or runtime image during commit"

assuming that is accurate.